backend/.eslintignore

```diff
@@ -6,3 +6,4 @@ out/
 public/
 next.config.js
 vite.config.ts
+firebase-admin.ts
```

Comment:
Is this needed?

```suggestion
```

Reply:
It raises like 11 linting errors related to type, so I thought it'd be fine to ignore it since it's just a config file that we'll likely not have to interact with again. I might be missing something though; is it important for the linter to read it?

Reply:
Oh interesting, what are the errors the linter is giving you?

Reply:
I actually just fixed it; we were using require() for some reason instead of import and then I just did some typecasting shenanigans for everything else. Sorry about that; just for future reference, is it best to have the linter cover everything, even if it's essentially just a config file?

Reply:
Ideally, any code that gets written by someone should be covered by the linter. However, generated code can be ignored (e.g. output files from a build command).